spec/factories/phenotypes.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :phenotype do
+    characteristic "Penis length"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.